src/template/src/button/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
```

Comment:
if we're going to be flattening our own widget filter structure, should this be under `/src/widgets/Button.tsx` or `/src/Button.tsx` rather than under a button folder?

Reply:
@tomdye I think a button folder makes sense as it contains a readme and the test?